src/v/cloud_storage/partition_manifest.h

```diff
@@ -303,6 +326,10 @@ class partition_manifest final : public base_manifest {
 
     model::ntp _ntp;
     model::initial_revision_id _rev;
+
+    // Tracker of memory for this manifest.
```

Comment:
Perhaps worth saying this is tracking `_segments` in particular, not literally the total memory of the manifest.

Reply:
Done